tests/py/dynamo/conversion/harness.py

```diff
@@ -55,13 +59,13 @@ def run_test(
         rtol,
         atol,
         check_dtype=True,
+        pyt_inputs=None,
```

Comment:
Why is `pyt_inputs` needed? Can you add doc to explain each argument so that other developers are easy to follow?

Reply:
pyt_inputs is needed for a specific test_case for test_arange where you provide direct scalar values instead of torch tensors.